### DISCUSSION TITLE
pixelated images when reviewing recording

### DISCUSSION THREAD

**Discussion by stamandr** [blakeblackshear/frigate#13934] [Ask A Question]
I get a number of these pixelated images when trying to review a recording

Anyone know how to fix this ?

![image](https://github.com/user-attachments/assets/0c581cb2-fc7d-47af-8ed2-4df45536446e)

**Comment by NickM-27** [comment]
is that while watching recordings or while scrubbing?

**Reply by stamandr** [reply]
This happens when I go to review > detections > then select the image.

**Reply by NickM-27** [reply]
it's likely because you don't have a recording available for that time so it is just showing the preview, but a video recording would make it a lot easier to see what is going on

**Reply by stamandr** [reply]
You're right no recording. I have a parallel system running 0.13.2-6476f8a. This one has a video. I'm noticing quite a bit more without a video on 0.14.1-f4f3cfa compared to 0.13.2-6476f8a. Configs are the same.

**Reply by stamandr** [reply]
any idea as to why one has video (0.13.2-6476f8a) and the other (0.14.1-f4f3cfa) doesn't? Like I said, configs are the same as well as all hardware. Looking into this further, I am missing many videos on 14.1

**Reply by NickM-27** [reply]
see https://github.com/blakeblackshear/frigate/discussions/14094#discussioncomment-10811439

**Comment by stamandr** [comment]
Honestly, you now have me confused.  Doesn't the settings below mean it should record ?

I have the following in my config.

    review:
      alerts:
        required_zones:
          - Driveway_Entire_Area_Zone
          - Driveway_Only_Zone
          - Driveway_Person_Zone
          - Driveway_Car_Zone

**Reply by NickM-27** [reply]
Yes, alerts. You said previously your issue is with detections

**Reply by stamandr** [reply]
I said I get pixelated images when trying to review a recording. When I go to review and select the image this is when I get the pixelated image

**Reply by NickM-27** [reply]
Yes, you specifically said detections. Does this happen when clicking on an alert?

**Reply by stamandr** [reply]
It happens on Alerts, Detections and Motion

![image](https://github.com/user-attachments/assets/bab47f2a-313e-47b7-92c0-dda5d5ea8014)

**Reply by NickM-27** [reply]
That part uses the preview which can be blurry, especially when your cameras detect a lot of motion

**Reply by stamandr** [reply]
so it is supposed to do this for Alerts, Detections and Motion?

**Reply by NickM-27** [reply]
Not sure what you mean by supposed to. The previews are limited bandwidth to reduce storage. In some cases especially when a camera is seeing constant activity it may be pixelated. 

If you click on an alert the full recordings should load

**Reply by stamandr** [reply]
It appears to be hit and miss. Some load others don't
